src/app.js

```diff
@@ -25,6 +25,10 @@ var OffsetToLineColumnConverter = require('./lib/offsetToLineColumnConverter')
 
 var examples = require('./app/example-contracts')
 
+// components
+var fileExplorer = require('_file-explorer')
```

Comment:
Hm, I think that using directories to group modules makes stuff much more visible. This grouping is lost if you can just `require("_modulename")` from everywhere.